modules/administration/pages/custom-channels.adoc

```diff
@@ -78,12 +78,16 @@ For more information, see xref:administration:tshoot-registertradrhel.adoc[].
 ====
 
 
+Before you start with creating a software repository with the {webui} ensure that needed extra metadata is available.
```

Comment:
```suggestion
A pre requirement is, that you have a valid repository. Please check before you start adding a software repository to {productname} with the {webui},  that needed metadata are available. 
```

A repository for me is, what we have on disk with packages + metadata.
The repository in SUSE Manager is only including or using this repo. We do not really create it.
The text above is still not perfect. I am open for more suggestions.

Reply:
I hope I got it right now.  Of course, I'm not sure whether my version actually is better...